cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,43 @@
-describe('', () => {
+describe('CRUD article', () => {
+  let user;
+  let articleData;
   before(() => {
-
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      articleData = generateArticle;
+    });
+    cy.login(user.email, user.username, user.password);
   });
 
-  it('', () => {
-
+  it('should create the article', () => {
+    cy.visit('/editor');
+    cy.contains('.nav-link', 'New Article').click();
+    cy.get('[placeholder="Article Title"]')
+      .type(articleData.title);
+    cy.get('[placeholder="What\'s this article about?"]')
+      .type(articleData.description);
+    cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(articleData.body);
+    cy.contains('.btn', 'Publish Article')
+      .click();
+    cy.url().should('contain', 'article/');
   });
+  it('should delete the article', () => {
```

Comment:
Similar to the create test, it would be beneficial to add more assertions in the delete test. After deleting the article, you could assert that the article no longer exists in the list of articles.